bloom-housing/bloom
# chore: move Programs step to come after Students

# Pull Request Template

## Issue Overview

This PR addresses #2221 


## Description

The programs step in the application submission process had been placed directly after ADA options. Now it's placed after household changes and students.

## How Can This Be Tested/Reviewed?

Apply to a listing using the online forms and make sure the programs show up right before the income verification step.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
